syllabiShare/views.py

```diff
@@ -8,11 +8,6 @@
 
 
 def about(request):
-    (template, context) = authenticate(request.user)
```

Comment:
Oh wait, and what will the user menu dropdown show if they're logged out

Reply:
If you look at the page I removed the nav bar